dipy/viz/ui.py

```diff
@@ -2297,65 +2254,18 @@ def __init__(self, font_size, position, file_select):
         self.file_type = ""
         self.file_select = file_select
 
-        self.text_actor = self.build_actor(position=position, font_size=font_size)
-
-        self.handle_events(self.text_actor.get_actor())
-
-        self.on_left_mouse_button_clicked = self.left_button_clicked
-
-    def build_actor(self, position, text="Text", color=(1, 1, 1), font_family='Arial',
-                    justification='left', bold=False, italic=False,
-                    shadow=False, font_size='14'):
-        """ Builds a text actor.
-
-        Parameters
-        ----------
-        text: string
-            The initial text while building the actor.
-        position: (float, float)
-            The text position (x, y) in pixels.
-        color: (float, float, float)
-            Values must be between 0-1 (RGB).
-        font_family: string
-            Currently only supports Arial.
-        justification: string
-            Text justification - left, right or center.
-        bold: bool
-            Whether or not the text is bold.
-        italic: bool
-            Whether or not the text is italicized.
-        shadow: bool
-            Whether or not the text has shadow.
-        font_size: int
-            The font size of the text in pixels.
-
-        Returns
-        -------
-        text_actor: :class:`TextBlock2D`
-            The base text actor.
-
-        """
-        text_actor = TextBlock2D()
-        text_actor.position = position
-        text_actor.message = text
-        text_actor.font_size = font_size
-        text_actor.font_family = font_family
-        text_actor.justification = justification
-        text_actor.bold = bold
-        text_actor.italic = italic
-        text_actor.shadow = shadow
-        text_actor.color = color
-
+        self.text_actor = TextBlock2D(position=position, font_size=font_size,
+                                      color=(0, 0, 0))
+        self.text_actor.actor.GetTextProperty().SetLineSpacing(1)
         if vtk.vtkVersion.GetVTKSourceVersion().split(' ')[-1] <= "6.2.0":
             pass
         else:
-            text_actor.actor.GetTextProperty().SetBackgroundColor(1, 1, 1)
-            text_actor.actor.GetTextProperty().SetBackgroundOpacity(1.0)
+            self.text_actor.actor.GetTextProperty().SetBackgroundColor(1, 1, 1)
```

Comment:
Same as above, can you have the `build_actor` or `_build` method do this?
(I guess we should stick to the `_build` name for the method as it's private)